src/components/directory-content.js

```diff
@@ -679,21 +694,29 @@ const DirectoryContent = () => {
         setIsMetadataLoading(true);
         if (currentChildren !== null && currentChildren.length > 0) {
             let metadata = {};
-            fetchElementsInfos(currentChildren.map((e) => e.elementUuid))
-                .then((res) => {
-                    res.forEach((e) => {
-                        metadata[e.elementUuid] = {
-                            name: e.elementName,
-                            subtype: e.specificMetadata
-                                ? e.specificMetadata.type
-                                : null,
-                        };
+            let childrenToFetchElementsInfos = Object.values(currentChildren)
+                .filter((e) => !e.uploading)
+                .map((e) => e.elementUuid);
+            if (
+                childrenToFetchElementsInfos !== null &&
```

Comment:
Object.values return value is an array, can not be null, even if passed null as parameter, as it would have thrown  Uncaught TypeError: can't convert null to object